frontend/src/lib/api/stocks.ts

```diff
@@ -0,0 +1,23 @@
+import { stocksDetails } from "$lib/stores";
+import { get } from "../request";
+
+interface TickerDetails {
+    ticker: string,
+    company_name: string,
+    exchange: string
+}
+
+async function getStocksDetails(): Promise<void> {
+    const tickersDetails = await get<TickerDetails[]>({ endpoint: "/stocks/all" });
+    const stocksDetailsMap = new Map<string, TickerDetails>();
```

Comment:
Also I believe this can be done in one line with something like `new Map(tickerDetails.map(x => [x.ticker, x])`;